### DISCUSSION TITLE
建议优化一下关于首页顶栏样式的插件，V2版本感觉还不如V1的好看灵活。

### DISCUSSION THREAD

**Discussion by Flying-Fish-18** [the1812/Bilibili-Evolved#3956] [功能建议 / Ideas]
### **V2版本：**
![image](https://user-images.githubusercontent.com/80080830/215434534-2a291fdf-770d-4a09-9ecd-ce1e8261781c.png)

![image](https://user-images.githubusercontent.com/80080830/215435008-aa6f0d26-98cc-40d8-b3b2-36fc6c0983dc.png)

### **V1版本：**
![image](https://user-images.githubusercontent.com/80080830/215434664-d624522d-a956-456b-a11a-c11e194ebc5b.png)

![image](https://user-images.githubusercontent.com/80080830/215435172-0a0667f9-a844-46ae-908e-95ba33666105.png)

**V1版本的自定义顶栏功能还可以排列顶栏那几个入口的顺序和是否隐藏，到了V2就都没有了。**

**Comment by Tinhone** [comment]
Commit: 3397e56dc
![图片](https://user-images.githubusercontent.com/81268244/215541202-c666969b-d6b0-4ba5-b403-b0dd5cd700cb.png)

**Comment by the1812** [comment]
一直都支持的
![image](https://user-images.githubusercontent.com/26504152/215629564-a8c083d2-cf0e-4726-9da3-f5e80aeb01fe.png)
